exercises/markdown/markdown.py

```diff
@@ -0,0 +1,71 @@
+import re
+
+
+def parse_markdown(markdown):
+    lines = markdown.split('\n')
```

Comment:
I think you forgot to replace an actual solution with an exercise placeholder for the learner.
Could you please fix this?

Reply:
Learner should refactor this code as I mentioned before.

Reply:
@PaulGlass yeah, I see it now, thanks